### ISSUE TITLE
Add Missing Methods for Environments

### ISSUE THREAD

**Issue by Cliftonz** [novuhq/novu-python#74]
Some endpoints of the following section can't be hit from this SDK. 

We would like to ensure that all endpoints can be hit. For this issue, we would like to build out the Environments

Some of these methods may already exist.  If so no need to change/update them, just point out that they already exist and we will close the issue.  This is just to bring every SDK we have on par with all the methods available.

Check all the endpoints [here](https://api.novu.co/api#/Workflow%20groups/NotificationGroupsController_updateNotificationGroup) and add the missing ones.

**Comment by mr-palindrome** [comment]
I looked for the methods that are available in SDK! The Put and Get requests (https://api.novu.co/api#/Environments) for the API Keys doesn't work as per requirement! 

I would like to work on implementing these two requests as it is intended in swagger and documentation!

**Comment by unicodeveloper** [comment]
Please use the API reference docs with your api keys https://docs.novu.co/api-reference/environments/get-current-environment

@mr-palindrome

**Comment by mr-palindrome** [comment]
> Please use the API reference docs with your api keys https://docs.novu.co/api-reference/environments/get-current-environment
> 
> @mr-palindrome

Hey @unicodeveloper , I checked that but there is no method for that in the python SDK there is one method api_keys, but it doesn't work as intended in the docs!

**Comment by unicodeveloper** [comment]
@mr-palindrome I am not sure I understand what you mean. Please can you rephrase?

**Comment by unicodeveloper** [comment]
I am looking through the code and I can actually see that we have Environments method for the following:

- Get environments
- Create environment
- Get Current environment
- Get API Keys

What is missing is:
- Update env by Id - https://docs.novu.co/api-reference/environments/update-env-by-id
- Regenerate API Keys - https://docs.novu.co/api-reference/environments/regenerate-api-keys

So, these are the two methods you need to work on adding to the Environments Class @mr-palindrome

**Comment by mr-palindrome** [comment]
> I am looking through the code and I can actually see that we have Environments method for the following:
> 
> * Get environments
> * Create environment
> * Get Current environment
> * Get API Keys
> 
> What is missing is:
> 
> * Update env by Id - https://docs.novu.co/api-reference/environments/update-env-by-id
> * Regenerate API Keys - https://docs.novu.co/api-reference/environments/regenerate-api-keys
> 
> So, these are the two methods you need to work on adding to the Environments Class @mr-palindrome

I might take time for update env method. But I would love to work on this!

**Comment by mr-palindrome** [comment]
hey @Cliftonz @unicodeveloper can you help me in understanding the updateMyEnvironment endpoint! I have also created a support ticket on discord(https://discord.com/channels/895029566685462578/1160432550301405276). It would be really helpful if anyone among you can help me to understand it!

**Comment by mr-palindrome** [comment]
Hey @Cliftonz you can close this issue now as #99 solves this issue!
Thanks for giving me the opportunity to work on this!
